### PR TITLE
ci: Prepare for 0.4.0-beta

### PR DESCRIPTION
**Pull request by pankajagrawal16**
**Issue #, if available:**

## Description of changes:

**Checklist**

 

* [x] [Meet tenets criteria](https://awslabs.github.io/aws-lambda-powertools-java/#tenets)
* [ ] Update tests
* [x] Update docs
* [x] PR title follows [conventional commit semantics]()

## Breaking change checklist

**RFC issue #**:

* [ ] Migration process documented
* [ ] Implement warnings (if it can live side by side)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/utilities/sqs_large_message_handling.mdx [hunk 1]
```diff
@@ -29,7 +29,7 @@ To install this utility, add the following dependency to your project.
 <dependency>
     <groupId>software.amazon.lambda</groupId>
     <artifactId>powertools-sqs</artifactId>
-    <version>0.3.1-beta</version>
+    <version>0.4.0-beta</version>
 </dependency>
 ```
```
Comment (jeromevdl) [inline docs/content/utilities/sqs_large_message_handling.mdx:34]: make me think I forgot to add this in the parameters doc (installation step): 
```
<dependency>
    <groupId>software.amazon.lambda</groupId>
    <artifactId>powertools-parameters</artifactId>
    <version>0.4.0-beta</version>
</dependency>
```

File: docs/content/index.mdx [hunk 2]
```diff
@@ -16,12 +16,12 @@ Powertools dependencies are available in Maven Central. You can use your favouri
...
-        <version>0.3.1-beta</version>
+        <version>0.4.0-beta</version>
     </dependency>
     <dependency>
         <groupId>software.amazon.lambda</groupId>
         <artifactId>powertools-logging</artifactId>
-        <version>0.3.1-beta</version>
+        <version>0.4.0-beta</version>
     </dependency>
     ...
```
Comment (jeromevdl) [inline docs/content/index.mdx:26]: metrics is not there? normal?

File: docs/content/utilities/sqs_large_message_handling.mdx (hunk 1 above)
Comment (pankajagrawal16) [reply docs/content/utilities/sqs_large_message_handling.mdx:34]: aha.. nevermind, I can fix it in same PR

File: docs/content/index.mdx (hunk 2 above)
Comment (pankajagrawal16) [reply docs/content/index.mdx:31]: Good catch. I guess we missed updating it when we launched metrics module 😉

**Review by jeromevdl — APPROVED** [review]
Good for me!

**Review by msailes — APPROVED** [review]
Great!

**Pull request merged**